### PR TITLE
Date utils test

### PR DESCRIPTION
**Pull request by sinasen2000**

### CODE REVIEW

File: core/src/test/java/nl/nn/adapterframework/util/DateUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+		assertEquals("15-01-16", date);
+	}
+
+	/**
+	 * Method: isSameDay(Date date1, Date date2)
+	 */
+	@Test
+	public void testIsSameDay() throws Exception {
+		Date d1 = DateUtils.parseAnyDate("10-10-2013");
+		Date d2 = DateUtils.parseAnyDate("2013-10-10");
```
Comment (nielsm5) [inline core/src/test/java/nl/nn/adapterframework/util/DateUtilsTest.java:109]: is it dd-mm-yyyy or mm-dd-yyyy?
Please use a different day and month

File: core/src/test/java/nl/nn/adapterframework/util/DateUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+		assertEquals("Thu Oct 10 00:00:00 CEST 2013", d.toString());
+	}
+
+	/**
+	 * Method: parseAnyDate(String dateInAnyFormat)
+	 */
+	@Test
+	public void testParseAnyDate() throws Exception {
+		Date d = DateUtils.parseAnyDate("10-2013-10");
+		assertEquals("Thu Oct 10 00:00:00 CEST 2013", d.toString());
```
Comment (nielsm5) [inline core/src/test/java/nl/nn/adapterframework/util/DateUtilsTest.java:52]: What's the dateformat here? You're actually testing the `CalendarParser` class here?

File: core/src/test/java/nl/nn/adapterframework/util/DateUtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+		assertEquals("1970-01-18 09:40:00.000", s);
+
+	}
+
+	/**
+	 * Method: parseToDate(String s, String dateFormat)
+	 */
+	@Test
+	public void testParseToDate() throws Exception {
+		Date d = DateUtils.parseToDate("10-10-2013", DateUtils.FORMAT_DATE);
```
Comment (nielsm5) [inline core/src/test/java/nl/nn/adapterframework/util/DateUtilsTest.java:33]: `FORMAT_DATE` is `dd-MM-yy` yet youre using yyyy maybe add more tests? does it ignore the last 2 `yy`?

**Review by nielsm5 — CHANGES_REQUESTED** [review]

File: core/src/test/java/nl/nn/adapterframework/util/DateUtilsTest.java (hunk 1 above)
Comment (sinasen2000) [reply core/src/test/java/nl/nn/adapterframework/util/DateUtilsTest.java:109]: Done

**Review by gvanbrakel — APPROVED** [review]

**Review by nielsm5 — APPROVED** [review]

**Pull request merged**